lib/manager/bitbucket-pipelines/extract.ts

```diff
@@ -0,0 +1,63 @@
+import { logger } from '../../logger';
+import { getDep } from '../dockerfile/extract';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile | null {
+  const deps: PackageDependency[] = [];
+
+  try {
+    const lines = content.split('\n');
+    for (let lineNumber = 0; lineNumber < lines.length; lineNumber += 1) {
+      const line = lines[lineNumber];
+
+      const pipeMatch = /^\s* - pipe:\s*'?"?([^\s'"]+)'?"?\s*$/.exec(line);
+      if (pipeMatch) {
+        const pipe = pipeMatch[1];
+        const pipeSplit = pipe.split(':');
+        const depName = pipeSplit[0].trim();
```

Comment:
Why you need to trim?

Reply:
To be sure thare are no spaces from both size. But I realized this case is impossible. Removed trim.